package.json

```diff
@@ -33,11 +33,9 @@
     "gh-pages": "^0.12.0",
     "gx": "^1.0.0-rc3",
     "jsdom": "^9.10.0",
-    "json-loader": "^0.5.4",
     "jsonp": "^0.2.1",
     "mocha": "^3.2.0",
     "mocha-jsdom": "^1.1.0",
-    "object-assign": "^4.1.0",
```

Comment:
`warning rebass@0.3.4: "dependencies" has dependency "object-assign" with range "^4.1.1" that collides with a dependency in "devDependencies" of the same name with version "^4.1.0"`